cordapp/src/main/kotlin/net/corda/plugins/CordappPlugin.kt

```diff
@@ -130,6 +141,16 @@ class CordappPlugin : Plugin<Project> {
         return Pair(targetPlatformVersion, minimumPlatformVersion)
     }
 
+    private fun parseVersion(versionStr: String?, project: Project): Int {
+        if (versionStr == null) return 1
+        return try {
+            Integer.parseInt(versionStr)
+        } catch (e: NumberFormatException) {
+            project.logger.info("Invalid version identifier: $versionStr. Defaulting to 1")
```

Comment:
It's most likely they meant something other than 1. Better to throw an exception explaining `versionStr` is not an _integer_ version.

Reply:
i.e. Gradle's `InvalidUserDataException`.

Reply:
Addressed.